package.json

```diff
@@ -177,6 +178,9 @@
     "testPathIgnorePatterns": [
       "<rootDir>/frontend/test/"
     ],
+    "moduleNameMapper": {
+      "\\.(css|less)$": "identity-obj-proxy"
```

Comment:
@attekei Turns out circle doesn't seem to like this package. @tlrobinson Added a simple mock as part of #4765 which seems to work just fine though.